.config/protractor.conf.js

```diff
@@ -22,15 +22,16 @@ exports.config = {
     showColors: true,
     isVerbose: false,
     includeStackTrace: false,
-    defaultTimeoutInterval: 5000000
+    defaultTimeoutInterval: 5000000,
+    print: function () {}
```

Comment:
why do you need this?

Reply:
"print: function () {}" - to remove points from default jasmine reporter